arches_querysets/models.py

```diff
@@ -387,7 +387,7 @@ def find_nodegroup_alias(self):
         if not getattr(self, "_nodegroup_alias", None):
             # But perform a last-minute check just in case. This will
             # also run if the node alias is null.
-            if grouping_node := Node.objects.first(pk=self.nodegroup_id):
+            if grouping_node := Node.objects.get(pk=self.nodegroup_id):
```

Comment:
To ensure v8 compatibility let's keep first:
```suggestion
            if grouping_node := Node.objects.filter(pk=self.nodegroup_id).first():
```